OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -497,23 +473,18 @@ void simulateMuscle(
         /*double xSpeed = */modelCoordinateSet[0].getSpeedValue(si);
         // double KEsysCheck = 0.5*ballMass*xSpeed*xSpeed;
         /*double PEsys =  */model.getMultibodySystem().calcPotentialEnergy(si);
-        double jointWork = jointWorkProbe->computeProbeInputs(si)(0);
-        double ESysMinusWork = Esys 
-                                - muscWorkProbe->computeProbeInputs(si)(0)
-                                - jointWork; 
-
+        double jointWork = jointWorkProbe->getProbeOutputs(si)(0);
```

Comment:
Based on discussions, I think this is where the energy was being compared but there's now a more rigorous test for power so I think this block can just be deleted.